TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Service/ImportExport/NodeImportService.php

```diff
@@ -512,6 +512,9 @@ protected function persistEntities($propertyValue) {
 	 */
 	protected function parseEndElement(\XMLReader $reader) {
 		switch ($reader->name) {
+			case 'hiddenBeforeDateTime':
+			case 'hiddenAfterDateTime':
+			case 'contentObjectProxy':
```

Comment:
I'm actually not sure about that one. I just checked in the export service what element names are created and added that. Not sure what it does though. I will keep that out in the next patch.